api/v1beta2/integrationtestscenario_types.go

```diff
@@ -45,6 +45,8 @@ type IntegrationTestScenarioSpec struct {
 	ResolverRef ResolverRef `json:"resolverRef"`
 	// Params to pass to the pipeline
 	Params []PipelineParameter `json:"params,omitempty"`
+	// ResolverType defines the type of resolver,default resolver type is pipeline
+	ResolverType string `json:"pipeline,omitempty"`
```

Comment:
I'm guessing this field isn't needed because of the `resolverType` field already being within the resolverRef struct [here](https://github.com/konflux-ci/integration-service/pull/1091/files#diff-75d2447d6230ec91af84b18efdf8da2f07d386a52830fc04dd0b2be6a605bf40R111)?

Reply:
hmm, how would it default to pipeline ?

Reply:
I think it's because of the ``json:"pipeline,omitempty"`` part.